libs/chain-signatures/contract/src/primitives/signature.rs

```diff
@@ -1,9 +1,36 @@
 use crate::crypto_shared;
 
-use crypto_shared::{derive_epsilon, SerializableScalar};
-use k256::Scalar;
+use crypto_shared::derive_tweak;
 use near_sdk::{near, AccountId, CryptoHash};
 
+#[derive(Debug, Clone, Eq, Ord, PartialEq, PartialOrd)]
+#[near(serializers=[borsh, json])]
+pub struct Tweak([u8; 32]);
```

Comment:
Since we're doing this, should we also change the json encoding to a hex string instead of a long list of integers? @bowenwang1996 I think introducing "hex" as a dependency should be fine - it's very little code, and the derived json code is likely not that much better.

Reply:
How does changing this struct help? Tweak is computed internally based on account id. It is not something given as an argument to a function